editor/src/components/canvas/ui-jsx-canvas-renderer/ui-jsx-canvas-component-renderer.tsx

```diff
@@ -194,12 +203,23 @@ export function createComponentRendererComponent(params: {
         lookupRenderer,
       )
 
-      definedWithinWithValues = runBlockUpdatingScope(
+      const definedWithinWithValues = runBlockUpdatingScope(
         params.filePath,
         mutableContext.requireResult,
         utopiaJsxComponent.arbitraryJSBlock,
         scope,
       )
+
+      spiedVariablesInScope = {
+        ...spiedVariablesInScope,
+        ...objectMap(
+          (spiedValue) => ({
+            spiedValue: spiedValue,
+            insertionCeiling: null,
```

Comment:
I know this is just moved code, but in line 139 above insertionCeiling is set to `instancePath`, shouldn't this also be `instancePath`? or in this case null and instancePath mean the same thing?

Reply:
In this case it doesn't make any particular difference from memory, but yes I don't want to change it in this PR, even though I think setting it to that would be beneficial even if just as a belt and braces change.